lib/runtime/wasmer/imports.go

```diff
@@ -1084,6 +1092,13 @@ func ext_default_child_storage_storage_kill_version_1(context unsafe.Pointer, ch
 	storage.DeleteChild(childStorageKey)
 }
 
+//export ext_default_child_storage_storage_kill_version_2
+func ext_default_child_storage_storage_kill_version_2(context unsafe.Pointer, a, b C.int64_t) C.int32_t {
+	logger.Debug("[ext_default_child_storage_storage_kill_version_2] executing...")
+	logger.Warn("[ext_default_child_storage_storage_kill_version_2] unimplemented")
+	return 0
```

Comment:
Do we need a TODO, or issue for this since it's unimplemented?

Reply:
I can open an issue for this